### PR TITLE
[CSL-1769] Use log-warper 1.3.3

### PR DESCRIPTION
**Pull request by int-index**
Since @volhovM told me upgrading to the latest `log-warper` would fix the issue, that's what I did. Well, the good news is that I finished the upgrade, the bad news is that it didn't fix the issue.

### CODE REVIEW

File: explorer/src/Pos/Explorer/Socket/Holder.hs [hunk 1]
```diff
@@ -83,9 +83,16 @@ mkConnectionsState =
 withConnState
     :: (MonadIO m, WithLogger m)
     => ConnectionsVar
-    -> NamedPureLogger (StateT ConnectionsState STM) a
+    -> StateT ConnectionsState (NamedPureLogger STM) a
     -> m a
-withConnState var = launchNamedPureLog $ atomically . modifyTVarS var
+withConnState var m = do
```
Comment (volhovM) [inline explorer/src/Pos/Explorer/Socket/Holder.hs]: mda

Comment (int-index) [reply explorer/src/Pos/Explorer/Socket/Holder.hs]: life is tough

**Comment by int-index** [comment]
Alright, https://github.com/serokell/log-warper/pull/27 fixes it! Thank you, @volhovM

**Comment by avieth** [comment]
No more stdout logging in csl master? Is this patch responsible for that? How can I get it back? @int-index @volhovM

**Comment by int-index** [comment]
The patch should've fixed the flag that disables logging. I only tested that logging is indeed disabled it when lcConsoleOutput is set to False, but not that logging still works when it's set to True...

**Comment by avieth** [comment]
I put `printOutput: true` in the top level of the configuration file but still only warnings and errors are printed.

**Comment by avieth** [comment]
```
topLevelParams :: [Text]
topLevelParams =
    ["rotation", "showTime", "printOutput", "filePrefix" ]
```

I think I need to set `termSeverity` to `Debug`, but I can't because it's not in the `topLevelParams`.

**Comment by int-index** [comment]
I created https://issues.serokell.io/issue/CSL-1778 to track this.

**Comment by int-index** [comment]
> I think I need to set `termSeverity` to `Debug`

Right, @volhovM told me the same, so I suppose that's what we should do.

**Pull request merged**